data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1300,6 +1300,10 @@ spec:
               azure:
                 description: Azure is the configuration used when installing on Azure.
                 properties:
+                  RHCOSImage:
```

Comment:
There's also a test in the explain pkg that needs to be changed. 
https://github.com/openshift/installer/blob/master/pkg/explain/printer_test.go